### PR TITLE
add routeSchemaOpts to exposeStatusRoute

### LINKED ISSUE #60: How to hide it's default route created by exposeStatusRoute:'/live' in swagger ui.
Screenshot from 2020-08-17 15-04-34

## Example

  fastify.register(async function (fastify) {
    fastify.register(underPressure, {

      maxEventLoopDelay: fastify.config.MAX_EVENT_LOOP_DELAY,
      maxHeapUsedBytes: fastify.config.MAX_HEAP_USED_BYTES,
      maxRssBytes: fastify.config.MAX_RSS_BYTES,

      exposeStatusRoute: "/live",
    });

### PR DESCRIPTION
**Pull request by climba03003**
Resolve #60 

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -81,7 +81,7 @@ async function underPressure (fastify, opts) {
       ...opts.exposeStatusRoute.routeOpts,
       url: opts.exposeStatusRoute.url,
       method: 'GET',
-      schema: {
+      schema: Object.assign({}, opts.exposeStatusRoute.routeSchemaOpts, {
```
Comment (mcollina) [inline index.js:84]: Can you add a test for this change?

**Review by mcollina — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (climba03003) [reply index.js:84]: Oh sorry, I only added the type check but forget the normal javascript checking

Comment (climba03003) [reply index.js:84]: test case added

**Comment by climba03003** [comment]
@mcollina test case added. do you have time to review again.

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**